sstables/sstables_manager.cc

```diff
@@ -111,7 +111,6 @@ storage_manager::config_updater::config_updater(const db::config& cfg, storage_m
             co_await sstm.update_config(cfg);
         });
     })
-    , observer(cfg.object_storage_config.observe(action.make_observer()))
```

Comment:
So observing the object_storage_config changed is dropped, but there's no place that observes changes on object_storage_endpoints instead

Reply:
Unless I miss something, [this](https://github.com/scylladb/scylladb/pull/22952#discussion_r1995083892) test should catch that endpoints update became broken